### ISSUE TITLE
Title of https://restic.readthedocs.io/en/latest/ is confusing

### ISSUE THREAD

**Issue by MichaelEischer** [restic/restic#4420]
The title of https://restic.readthedocs.io/en/latest/ is `Restic Documentation - restic 0.15.2 documentation` which suggests that this is the documentation for the latest released restic version. However, that documentation is located at https://restic.readthedocs.io/en/stable/ .

In addition to the misleading title, the documentation currently lacks an easily visible indication of the current version. The version selector in the lower left is too subtle. This has been a constant source of confusion for users, see https://forum.restic.net/t/confusing-wrong-documentation-on-group-by-flag/6494/4 for the most recent example.

**Comment by AdrianMastronardi** [comment]
One idea to reduce the “I’m reading the wrong version” confusion is to enable Read the Docs’ **Addons → Notifications** (version warning notifications).

RTD can show a prominent banner when the reader is **not on `stable`** (and specifically when they are on **`latest`**, which typically tracks the default branch / development docs). This would make it much clearer that `/en/latest/` is *development/unreleased* docs and that the recommended docs are `/en/stable/`.

Godot’s documentation is a good example of the UX we want here (see attached screenshot): when you land on `latest`, you immediately get a visible “this is unstable / here be dragons” warning with a nudge towards the stable docs.

If this sounds reasonable, we could:
- Enable **Non-stable** and/or **Latest** notifications in the RTD project settings (Settings → Addons → Notifications).
- Make the banner copy explicit: **`stable` = latest release** and **`latest` = development/unreleased** (may differ from the current release).

<img width="2427" height="783" alt="Image" src="https://github.com/user-attachments/assets/f80765a4-60da-437a-9bc1-16ec91106173" />

**Comment by MichaelEischer** [comment]
Right now when accessing the `latest` version of the docs there is the following notification:

<img width="683" height="157" alt="Image" src="https://github.com/user-attachments/assets/1088e27c-c6f1-49ba-aa2e-97d68cab0c42" />

I've also checked in the RTD settings, there the notifications you've mentioned are enabled. That more prominent banner must originate somewhere else. Maybe some other version of the documentation theme could offer that?

**Comment by AdrianMastronardi** [comment]
Isn't that sufficient to consider closing this issue?

**Comment by MichaelEischer** [comment]
True, let's hope this is sufficient and if not, we can revisit this topic again.
